huaweicloud/services/apig/resource_huaweicloud_apig_instance.go

```diff
@@ -369,12 +369,12 @@ func parseInstanceAvailabilityZones(azStr string) []string {
 }
 
 // The response of ingress acess does not contain EIP ID, just the IP address.
-func parseInstanceIngressAccess(config *config.Config, region, publicAddress string) (*string, error) {
+func parseInstanceIngressAccess(c *config.Config, region, publicAddress string) (*string, error) {
```

Comment:
better to keep it same with other funcs
```suggestion
func parseInstanceIngressAccess(cfg *config.Config, region, publicAddress string) (*string, error) {
```

Reply:
fixed